ownCloudTests/File List/MoreViewTests.swift

```diff
@@ -0,0 +1,114 @@
+
+			//Asserts
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.openin")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.move")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.rename")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.duplicate")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.copy")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.delete")).assert(grey_sufficientlyVisible())
+
+			//Reset status
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("dimming-view")).perform(grey_tap())
+			EarlGrey.select(elementWithMatcher: grey_allOf([grey_accessibilityLabel("Back"), grey_accessibilityTrait(UIAccessibilityTraits.staticText)])).perform(grey_tap())
+		} else {
+			assertionFailure("File list not loaded because Bookmark is nil")
+		}
+	}
+
+	/*
+	* PASSED PASSED if: MoreView is shown with folder options
+	*/
+	func testShowMoreViewForFolder() {
+
+		if let bookmark: OCBookmark = UtilsTests.getBookmark() {
+			//Mocks
+			OCMockSwizzlingFileList.mockOCoreForBookmark(mockBookmark: bookmark)
+			OCMockSwizzlingFileList.mockQueryPropfindResults(resourceName: "PropfindResponse", basePath: "/remote.php/dav/files/admin", state: .contentsFromCache)
+			self.showFileList(bookmark: bookmark)
+
+			//Actions
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("Documents-actions")).perform(grey_tap())
+
+			//Asserts
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.openin")).assert(grey_notVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.move")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.rename")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.duplicate")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.copy")).assert(grey_sufficientlyVisible())
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("com.owncloud.action.delete")).assert(grey_sufficientlyVisible())
+
+			//Reset status
+			EarlGrey.select(elementWithMatcher: grey_accessibilityID("dimming-view")).perform(grey_tap())
+			EarlGrey.select(elementWithMatcher: grey_allOf([grey_accessibilityLabel("Back"), grey_accessibilityTrait(UIAccessibilityTraits.staticText)])).perform(grey_tap())
+		} else {
+			assertionFailure("File list not loaded because Bookmark is nil")
+		}
+	}
+
+	func showFileList(bookmark: OCBookmark, issue: OCIssue? = nil) {
```

Comment:
This code is duplicated in some classes. Is it possible to avoid code duplication for this method?

Reply:
I were thinking about it. Probably moving it to "UtilsTests"